internal/template-validator/tlsinfo/tlsinfo.go

```diff
@@ -192,8 +193,8 @@ func (ti *TLSInfo) CreateTlsConfig() *tls.Config {
 	}
 
 	if !ti.sspTLSOptions.IsEmpty() {
-		tlsConfig.CipherSuites = ti.sspTLSOptions.CipherIDs(nil)
-		minVersion, err := ti.sspTLSOptions.MinTLSVersionId()
+		tlsConfig.CipherSuites = common.CipherIDs(ti.sspTLSOptions.OpenSSLCipherNames, &logger.Log)
+		minVersion, err := crypto.TLSVersion(ti.sspTLSOptions.MinTLSVersion)
```

Comment:
This is not correct, because `crypto.TLSVersion()` function expects one of these names:
https://github.com/kubevirt/ssp-operator/blob/9e99e78cceda599b72055610d9ddb9575af95094/vendor/github.com/openshift/library-go/pkg/crypto/crypto.go#L37-L42

So a string like:  `VersionTLS12`, but `ti.sspTLSOptions.MinTLSVersion` is a string returned by this function:
https://github.com/kubevirt/ssp-operator/blob/11c2e890e986955b25ce7fd43790c4416e9c4bbb/internal/common/crypto_policy.go#L119-L134

For example: `1.2`.